mars/deploy/oscar/session.py

```diff
@@ -1697,6 +1701,27 @@ def __exit__(self, *_):
         self.close()
 
 
+async def _execute_show_progress(
```

Comment:
Rename this method as `_execute_with_progress`.

Reply:
Fixed.